VMBackup/main/fsfreezer.py

```diff
@@ -287,6 +290,7 @@ def thaw_safe(self):
             self.log_binary_output()
             self.logger.log(error_msg, True, 'Error')
         self.logger.enforce_local_flag(True)
+        self.releaseFileLock()
```

Comment:
can this be under finally ? we may not need to add in different places then

Reply:
but there is no try block in thaw so do we add a try block as well? and we are calling it in the single place. under skip freeze we don't need that call